ui/src/components/PinPreview.tsx

```diff
@@ -72,7 +72,7 @@ export function PinPreview({
           Read Full Story
         </StyledButton>
         <CardDetailText>
-          {author} • {date}
+          {author} • {date.substring(0, 4)}
```

Comment:
[nit] if we're getting the `createdAt` value as a typescript `Date` object can we use `dateObj.getYear()` instead?

Reply:
I think I'm just going to revert it entirely since Deep is changing it in the database to just year so it can just fetch it as a string